### PR TITLE
Update seed database to 2.2.0 (gene sets) + gsva test data

### PR DESCRIPTION
**Pull request by sandertan**
This PR includes:
1. Seed-DB migrated to 2.1.0
2. `gbm_tcga` test data
GSVA score and p-value staging files are calculated for the microarray expression data of the glioblastoma provisional study. Additions and changes:
    - Data & meta file for GSVA score
    - Data & meta file for GSVA pvalue
    - Case list for GSVA study (same as case list for microarray expression)
    - Modification to `meta_expression_Zscores.txt` to include `source_stable_id: mrna_U133`
    - `gene_set` folder contains:
        - ⚠️ GMT file with gene sets used in this study, can be imported with `importGenesetData.pl`. Required before importing test study.
        - Hierarchical tree (`.yaml`) with gene sets used in this study, necessary for pop-up on query page. Can be imported with `importGenesetHierarchy.pl`.
        - R Markdown with method how GSVA scores and pvalues are calculated.

### CODE REVIEW

**Comment by sandertan** [comment]
2.1.0 database schema has been updated a few days ago, so this seed database is out of date. Will update soon.

**Comment by sandertan** [comment]
Database schema 2.1.0 has been updated recently (https://github.com/cBioPortal/cbioportal/pull/2155), so in latest commit seed database has been updated to this version. This seed does not contain gene sets. GMT file with gene sets used for test study is included in `gbm_tcga` staging files folder `gene sets`, and can be imported with `importGenesetData.pl`.

File: seedDB/README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 :information_source: the DB schema and the cBioPortal can follow different numbering cycles. This means that the version numbers won't be always identical. 
 
-You can download the files by using the links below:
-
-- **Schema 1.3.0**: [SQL file with create table statements for portal release 1.3.1](https://raw.githubusercontent.com/cBioPortal/cbioportal/v1.3.1/core/src/main/resources/db/cgds.sql) 
-- **Seed data, part1**: [cbioportal-seed SQL (.gz) file - part1 (no pdb_ tables)](https://github.com/cbioportal/datahub/raw/b69c86803c40d543080bf31a645721d06c82d08d/seedDB/seed-cbioportal_no-pdb_hg19.sql.gz)
-- **Seed data, part2 (optional)** [cbioportal-seed SQL (.gz) file - part2 (only pdb_ tables)](https://github.com/cbioportal/datahub/raw/b69c86803c40d543080bf31a645721d06c82d08d/seedDB/seed-cbioportal_only-pdb.sql.gz)
+You can download the schema by using the link below:
 
+- **Schema 2.1.0**: [SQL file with create table statements for portal release 2.1.0](https://github.com/cBioPortal/cbioportal/blob/geneset_frontend/db-scripts/src/main/resources/cgds.sql)
```
Comment (pieterlukasse) [inline seedDB/README.md]: change to release link

File: seedDB/README.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
-You can download the files by using the links below:
-
-- **Schema 1.3.0**: [SQL file with create table statements for portal release 1.3.1](https://raw.githubusercontent.com/cBioPortal/cbioportal/v1.3.1/core/src/main/resources/db/cgds.sql) 
-- **Seed data, part1**: [cbioportal-seed SQL (.gz) file - part1 (no pdb_ tables)](https://github.com/cbioportal/datahub/raw/b69c86803c40d543080bf31a645721d06c82d08d/seedDB/seed-cbioportal_no-pdb_hg19.sql.gz)
-- **Seed data, part2 (optional)** [cbioportal-seed SQL (.gz) file - part2 (only pdb_ tables)](https://github.com/cbioportal/datahub/raw/b69c86803c40d543080bf31a645721d06c82d08d/seedDB/seed-cbioportal_only-pdb.sql.gz)
+You can download the schema by using the link below:
 
+- **Schema 2.1.0**: [SQL file with create table statements for portal release 2.1.0](https://github.com/cBioPortal/cbioportal/blob/geneset_frontend/db-scripts/src/main/resources/cgds.sql) 
+- **Seed data, part1**: [cbioportal-seed SQL (.gz) file - part1 (no PDB tables)](seed-cbioportal_no-pdb_hg19.sql.gz)
```
Comment (pieterlukasse) [inline seedDB/README.md]: wrong link (missing version nr)

File: seedDB/README.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+- **Seed data, part2 (optional)** [cbioportal-seed SQL (.gz) file - part2 (only PDB tables)](seed-cbioportal_only-pdb.sql.gz)
```
Comment (pieterlukasse) [inline seedDB/README.md]: wrong link (missing version nr)

File: seedDB/README.md (hunk 1 above)
Comment (pieterlukasse) [inline seedDB/README.md]: probably release 1.6 or 1.7....not 2.something...

File: seedDB/README.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
 These files are MySQL database dump for seeding a new instance of the cBioPortal DB. They contain all the necessary background data for a properly functioning cBioPortal website, including cancer types, gene, uniprot-mappings, drug, pdb and network data. Data is compatible with:
 
-- DB schema version **1.3.0**
+- DB schema version **2.1.0**
 
 Which is suitable for:
 
-- release version **1.3.1** of cBioPortal.  
-- release version **> 1.3.1** of cBioPortal (:warning: with possible migration script step). 
+- release version ~~**1.3.1**~~ of cBioPortal.
```
Comment (pieterlukasse) [inline seedDB/README.md]: probably 1.5.0 now??

**Comment by sandertan** [comment]
I replaced this PR with https://github.com/cBioPortal/datahub/pull/45 and https://github.com/cBioPortal/datahub/pull/44

**Pull request closed without merging**